src/services/opensrp/index.ts

```diff
@@ -58,6 +59,19 @@ export function getPayload(method: HTTPMethod) {
   };
 }
 
+/** get payload for fetch
+ * @param {AbortSignal} signal - signal object that allows you to communicate with a DOM request
+ * @param {HTTPMethod} method - the HTTP method
+ * @returns the payload
+ */
+/* tslint:disable-next-line: no-unused-variable */
+export function getPayloadOptions(signal: AbortSignal, method: HTTPMethod) {
+  return {
+    headers: getDefaultHeaders() as HeadersInit,
+    method,
+  };
+}
```

Comment:
I think we can remove the tslint disable and simple change unused variables to `_` like: 

```ts
getPayloadOptions(_: AbortSignal, method: HTTPMethod) { ... }
```